examples/complete/main.tf

```diff
@@ -64,7 +64,7 @@ module "slz_vpc" {
   region            = var.region
   prefix            = var.prefix
   tags              = var.resource_tags
-  name              = "${var.prefix}-vpc"
+  name              = "vpc"
```

Comment:
was duplicating prefix name, resulting in name being too long.

Reply:
how about just setting it to var.prefix ?

Reply:
It's coming from here https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/blob/118d05d9d51c4097a25d4bd17336e268b62c64de/main.tf#L104